components/Snackbar/src/MDCSnackbarManager.m

```diff
@@ -46,6 +46,12 @@ static BOOL UIViewHasFocusedAccessibilityElement(UIView *view) {
  */
 @interface MDCSnackbarManagerInternal : NSObject
 
+/**
+ The instance of MDCSnackbarManager that "owns" this internal manager.  Used to get theming
+ properties. Can be refactored away in the future.
```

Comment:
It looks as though the entire "internal" class can be merged into `MDCSnackbarManager`. I believe it only ever existed to avoid making the singleton explicit in the first place.